File: codeabbey/006-rounding.js
#!/usr/bin/env node

var input, output;

input = require('fs')
        .readFileSync('./input.txt', 'utf8')
        .split('\n')
        .slice(1, -1)
        .map(function(nums) {
          return nums.split(' ')
                .map(function(num) {
                  return parseInt(num, 10);
                });});

output = input.map(function(nums) {
  var num = nums[0] / nums[1];
  return Math.round(num);
}).join(' ');

console.log(output);

File: scratchpad/ruby/codeabbey/006_rounding.rb
#!/usr/bin/env ruby
# encoding: utf-8

x = ARGF.file.read.split("\n").map do |e|
  nums = e.split(' ')
  nums.size == 2 ? (nums[0].to_f / nums[1].to_f).round : nil
end.compact.join ' '
puts x
